docs/developer-docs/setup/hello-world.mdx

```diff
@@ -0,0 +1,342 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Hello, world!
+
+This guide shows how to get started developing on ICP quickly by deploying your first 'Hello, world!' smart contract. 
+
+## Install dependencies
+
+### Install the IC SDK (v0.15.2 and newer)
+
+Download and install the latest version of [dfx](install/index.mdx). dfx is a CLI tool used for developing and deploying ICP smart contracts:
+
+```bash
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+```
+
+### Install [Node.js](https://nodejs.org/en/download)
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+nvm install 18
+```
+
+## Start the local replica
+
+```bash
+dfx start
+```
+
+This command starts a local instance of the ICP replica using a single node testnet. ICP does not have a public testnet for deployment. The output will return a dashboard URL that can be used to view information about the local replica:
+
+```
+Running dfx start for version 0.15.2
+Initialized replica.
+Dashboard: http://localhost:35375/_/dashboard
+```
+
+:::info
+While `dfx start` is running, you will not be able to run other commands in the terminal window. You can open another window, or start dfx in the background with the `dfx start --background` flag. 
+:::
+
+
+## Create a new project
+
+<Tabs groupId="language">
+<TabItem value="motoko" label="Motoko" default>
+
+```bash
+dfx new hello
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="rust" label="Rust">
+
+```bash
+dfx new hello --type=rust
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="TypeScript">
+
+```bash
+npx azle new azle_hello
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+
+Create the project directory and file structure:
+
+```bash
+mkdir kybra_hello
+cd kybra_hello
+
+mkdir src
+
+touch src/main.py
+touch dfx.json
+```
+
+Create a virtual environment:
+
+```bash
+~/.pyenv/versions/3.10.7/bin/python -m venv venv
+source venv/bin/activate
+```
+
+Install Kybra:
+
+```bash
+pip install kybra
+```
+
+</TabItem>
+</Tabs>
+
+For projects created with `dfx new` (Motoko and Rust) the command automatically generates the project's default configuration and two default smart contracts. 
```

Comment:
Is it possible to push this inside the Motoko and Rust tabs? Otherwise it is confusing for Azle and Kybra. If that's impossible, then let's remove this paragraph.

Reply:
Moved into Rust/Motoko tabs